tox.ini

```diff
@@ -236,7 +248,9 @@ commands =
     python {toxinidir}/system_tests/attempt_system_tests.py {posargs}
 deps =
     {[testing]deps}
-passenv = CI_* CIRCLE* GOOGLE_* TRAVIS* encrypted_*
+passenv =
+    {[testing]passenv}
+    encrypted_*
```

Comment:
System tests. These are env. vars. set automatically by the Travis CLI too. See https://github.com/GoogleCloudPlatform/google-cloud-python/blob/f348615f6e1eec92cedbbe4ea1327ba27d7100ee/system_tests/attempt_system_tests.py#L74-L76 and the module docstring in the same file